src/main.ts

```diff
@@ -1,8 +1,18 @@
 import Vue from "vue";
 
+declare module "vue/types/vue" {
```

Comment:
Maybe better to move that to `typing.d.ts`

Reply:
Thanks a good idea, thanks